schemas/syslog.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: syslog
+  title: Syslog
+  description: >
+    Source fields describe details about the source of a packet/event.
+    Source fields are usually populated in conjunction with destination fields.
+  type: group
+  fields:
+
+    - name: facility
+      level: core
+      type: long
+      example: 1
+      description: >
+        Value parsed from syslog messages. It represents the process
+        the message is originated from.
+
+    - name: facility_label
+      level: extended
+      type: keyword
+      description: >
+        Human readable format of `syslog.facility`.
+
+    - name: priority
+      level: core
+      type: long
+      example: 1
+      description: >
+        Value parsed from syslog messages. It is calculated using the
+        formula `8 * syslog.facility + log.level`.
```

Comment:
We could shorten this a bit. ECS doesn't really need to elaborate on the formula. We're just telling people where to stick their Syslog `priority`, they don't need to calculate it in their pipeline, I would assume :-)